pub/index.html

```diff
@@ -0,0 +1,86 @@
+<!doctype html>
+<html class="no-js" lang="">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Maddie's Blog</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <link rel="apple-touch-icon" href="apple-touch-icon.png">
+        <!-- Place favicon.ico in the root directory -->
+
+        <link rel="stylesheet" href="css/normalize.css">
+        <link rel="stylesheet" href="css/main.css">
+        <script src="js/vendor/modernizr-2.8.3.min.js"></script>
+    </head>
+    <body>
+        <header>
+            <img style="width:100%;" src="images/banner.jpg">
+        </header>
+        <div style="width:1060px; margin: 0 auto;">
```

Comment:
So, by _embedded styles_ what I meant were CSS rules and properties contained in a `<style>` tag. These are _inline styles_, CSS properties applied directly to an element with the HTML `style` property. There are important differences. Mostly, as you've pointed out, inline styles are really annoying.